common/src/main/java/com/revenuecat/purchases/common/GoogleOfferingParser.kt

```diff
@@ -1,7 +1,6 @@
-package com.revenuecat.purchases.google
+package com.revenuecat.purchases.common
```

Comment:
While this class is for Google and ideally it should live in the `feature/google` module, that made it significantly harder to keep the new classes in the `common` module. I could have moved them to `purchases`, but I kinda want to keep as much code as possible in `common`. 

Without this, I would have had to mock the offerings parser in the common module tests, and it felt those test should use the real implementation (as they were doing until now)